autoload/iced/buffer.vim

```diff
@@ -105,11 +105,13 @@ endfunction
 
 function! s:scroll_to_bottom(nr, _) abort
   let current_window = winnr()
+  let last_window = winnr('#')
   try
     let &eventignore = 'WinEnter,BufEnter'
     call s:focus_window(bufwinnr(a:nr))
     silent normal! G
   finally
+    call s:focus_window(last_window)
```

Comment:
Could you add a comment why this `focus_window` is needed because we will forget it later?

Reply:
Sure, it's done.